firewall replace-rules: panic: runtime error: index out of range [0] with length 0

**Issue description:**
hcloud firewall replace-rules is throwing the following error:
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/hetznercloud/cli/internal/cmd/firewall.runFirewallReplaceRules(0xc0000fa000, 0xc00017fb80, 0xc000178ed0, 0x1, 0x3, 0x0, 0x0)
        /Users/runner/work/cli/cli/internal/cmd/firewall/replace_rules.go:87 +0xc91
github.com/hetznercloud/cli/internal/state.(*State).Wrap.func1(0xc00017fb80, 0xc000178ed0, 0x1, 0x3, 0x0, 0x0)
        /Users/runner/work/cli/cli/internal/state/helpers.go:26 +0x56
github.com/spf13/cobra.(*Command).execute(0xc00017fb80, 0xc00001e210, 0x3, 0x3, 0xc00017fb80, 0xc00001e210)
        /Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:850 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xc00009a840, 0xc00009a840, 0x0, 0xc0000fa0d0)
        /Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:895
main.main()
        /Users/runner/work/cli/cli/cmd/hcloud/main.go:37 +0x73

**Version(s) affected**
Version 1.21.0 and 1.21.1

**Steps to reproduce:**
1. Create rules fiile fwtest.json with the following content:
[
      {
         "direction": "in",
         "source_ips": [
            "1.1.1.1/32"
         ],
         "destination_ips": [],
         "protocol": "tcp",
         "port": "443"
      }
]
2. Create firewall
hcloud firewall create --name "fw-test" --rules-file ./fwtest.json
3. Replace rules with the same file
hcloud firewall replace-rules fw-test --rules-file ./fwtest.json

--- a/internal/cmd/firewall/replace_rules.go
+++ b/internal/cmd/firewall/replace_rules.go
@@ -69,7 +69,7 @@ func runFirewallReplaceRules(cli *state.State, cmd *cobra.Command, args []string
 		}
 		switch d {
 		case hcloud.FirewallRuleDirectionOut:
-			r.DestinationIPs = make([]net.IPNet, 0, len(rule.DestinationIPs))
+			r.DestinationIPs = make([]net.IPNet, len(rule.DestinationIPs))
 			for i, ip := range rule.DestinationIPs {
 				_, n, err := net.ParseCIDR(ip)
 				if err != nil {
@@ -78,7 +78,7 @@ func runFirewallReplaceRules(cli *state.State, cmd *cobra.Command, args []string
 				r.DestinationIPs[i] = *n
 			}
 		case hcloud.FirewallRuleDirectionIn:
-			r.SourceIPs = make([]net.IPNet, 0, len(rule.SourceIPs))
+			r.SourceIPs = make([]net.IPNet, len(rule.SourceIPs))
 			for i, ip := range rule.SourceIPs {
 				_, n, err := net.ParseCIDR(ip)
 				if err != nil {

